From 231eb4c75fbfaaf90bca7702bdaf55cf701a3527 Mon Sep 17 00:00:00 2001
From: Alistair Francis <alistair.francis@wdc.com>
Date: Mon, 23 Jul 2018 11:23:39 -0700
Subject: [PATCH] jemalloc/jemalloc.cfg: Specify the host when building
 jemalloc

To avoid this error:
    configure: error: cannot run C compiled programs.
    If you meant to cross compile, use `--host'.
when cross compiling specify the host when configuring jemalloc.

Signed-off-by: Alistair Francis <alistair.francis@wdc.com>
Upstream-Status: Inappropriate [configuration]
---
 src/jemalloc/jemalloc.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/jemalloc/jemalloc.cfg b/src/jemalloc/jemalloc.cfg
index 196129de6..79ebbab1c 100644
--- a/src/jemalloc/jemalloc.cfg
+++ b/src/jemalloc/jemalloc.cfg
@@ -3,3 +3,4 @@
 --with-private-namespace=je_vmem_
 --disable-xmalloc
 --disable-munmap
+--host=${HOST_SYS}
-- 
2.14.4

